### PR TITLE
Notifications system + Live notification updates on BFF & Console

### PR DESCRIPTION
**Pull request by salman-codewar**
Demo:
notification

### CODE REVIEW

**Review by Brackleycassinga — CHANGES_REQUESTED** [review]
Just few comments .Great work (👍🏾

File: builder/deploy_orgs_config.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   "mailer": {
+    "host": "smtp.gmail.com",
+    "port": 465,
+    "username": "smtp",
+    "password": "smtp_password"
+   }
+  },
+  {
+   "name": "Salman",
+   "type": "empowerment",
```
[the remark this answers is not in the document]
Comment (salman-codewar) [reply builder/deploy_orgs_config.json:24]: This is related to the new script i added `deploy_orgs`. We never did drop this concept. Our global systems runs under it.
Added this script to test the console locally in future on these different behaviours. Specially invitation flow.

File: systems/data-plan/docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ services:
  postgresd-dataplan:
   image: postgres:13.3
   ports:
-   - '5404:5432'
+   - 5404:5432
```
[the remark this answers is not in the document]
Comment (salman-codewar) [reply systems/data-plan/docker-compose.yml:6]: Cleaned up these docker compose in a support for my `deploy_orgs` script. This change is needed.

File: systems/data-plan/docker-compose.yml [hunk 3]
```diff
@@ -79,7 +79,7 @@ services:
    context: ./api-gateway
   restart: always
   ports:
-   - '8074:8080'
+   - 8074:8080
```
[the remark this answers is not in the document]
Comment (salman-codewar) [reply systems/data-plan/docker-compose.yml:82]: This change is needed.

File: systems/notification/docker-compose.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ services:
  postgresd-notification:
   image: postgres:13.3
   ports:
-   - '5632:5432'
+   - 5632:5432
```
[the remark this answers is not in the document]
Comment (salman-codewar) [reply systems/notification/docker-compose.yml:7]: This change is needed.

**Review by Brackleycassinga — APPROVED** [review]

**Pull request merged**